### PR TITLE
Update join dataset rather than creating a new one

### PR DESCRIPTION
**Pull request by tracivar**
- Make a PUT request to update the existing join dataset with the dataset_column rather than creating a new join dataset and replace the existing one. This will be the same behaviour as new collection creation.

### CODE REVIEW

File: zeg/imagesets.py [hunk 1]
```diff
@@ -195,8 +195,17 @@ def _update_join_dataset(
     dataset_create_url = "{}datasets/".format(
         http.get_api_url(configuration["url"], configuration["project"])
     )
-    log.debug('POST: {}'.format(dataset_create_url))
```
Comment (exhaustedbrit) [inline zeg/imagesets.py:198]: Why did you remove this?

Comment (tracivar) [reply zeg/imagesets.py:198]: Because there's no post request here
